File: First Training C/my_strcat/my_strcat.c
/*
** ETNA PROJECT, $2022
** $MY_STRCAT
** File description:
** 	affiche str ?
*/

int my_strlen(const char *str) {   
    int y = 0;
    while ( str[y]) {
         y++;
    }
    return y;
}

char *my_strcat(char *dest, const char *src) {

    int i;
    int len = my_strlen(dest);

    for (  i = 0; src[i]; i++) {

        dest[len + i] = src[i];
    }
    dest[len + i] = '\0';
    return dest;
}

File: Typescript Fundamentals 2/my_strcat/my_strcat.ts
export function FindLenght(str:string){
  let i:number = 0;
  while(str[i]){
    i++
  }
  return i;
}
export function my_strcat(dest: string, src: string, nb:number){
  let j = FindLenght(dest)+1  
  let i=0
    while(i < nb){
    dest += src[i] 
    i++
    j++
    }
    return dest;
    
  
  }

